apps/class-solid/src/components/Experiment.tsx

```diff
@@ -101,7 +102,17 @@ export function ExperimentCard(experiment: Experiment) {
         <CardTitle>{experiment.name}</CardTitle>
         <CardDescription>{experiment.id}</CardDescription>
       </CardHeader>
-      <CardContent>{experiment.description}</CardContent>
+      <CardContent>
+        {experiment.description}
+        <Show when={experiment.output} fallback={<p>No output</p>}>
+          <details>
+            <summary>Output</summary>
+            <pre class="w-full h-60 overflow-auto">
+              {JSON.stringify(experiment.output, undefined, 2)}
+            </pre>
+          </details>
+        </Show>
```

Comment:
I'm not so keen on showing output here, it's fine for development but rather remove it before merge. Instead we could make it so that an analysis card is automatically created when the first experiment is added. And/or show progress/status of run instead.

```suggestion

```

Reply:
ok, removed